ui/js/controllers/perf/compare.js

```diff
@@ -58,6 +59,66 @@ perf.controller('CompareChooserCtrl', [
                 $scope.newRevision = tip;
             };
 
+            $scope.getPreviousRevision = function() {
+                $scope.proposalRevision = $scope.newRevisionError = null;
+
+                // only check for a full revision
+                if ($scope.newRevision.length != 12) return;
+
+                // indicate that we are loading if it takes some time
+                var timeout = setTimeout(function() {
+                    $scope.proposalRevisionLoading = true;
+                }, 400);
+
+                var promise;
+                if ($scope.newProject.name == "try") {
+                    // try require some special logic
+                    var iProjs = _.filter($scope.projects, function(proj) {
+                        return proj.name == "mozilla-inbound" ||
+                            proj.name == "mozilla-central" ||
+                            proj.name == "fx-team";
```

Comment:
We have a value called "thPerformanceBranches" in `ui/js/values.js`. Might be nice to use that here instead of hardcoding branch names (you'll need to inject it as a dependency)